From c8e0455ba9311d49ad6396f4fe9e39ca1584dbd3 Mon Sep 17 00:00:00 2001
From: Jqsfree <160339952+Jqsfree@users.noreply.github.com>
Date: Wed, 22 Jan 2025 01:36:29 +0800
Subject: [PATCH] Update README.md

---
 README.md | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/README.md b/README.md
index dc8ccd9..09e4409 100644
--- a/README.md
+++ b/README.md
@@ -1,2 +1,4 @@
-# hekko-world
-“此存储库用于练习 GitHub 流”
+# hekko-world  
+“此存储库用于练习 GitHub 流”  
+ 增加三个语言的hello-world
+ 
